src/storage/src/hummock/compactor/mod.rs

```diff
@@ -677,9 +678,24 @@ pub fn start_shared_compactor(
                                         .report_compaction_task(report_compaction_task_request)
                                         .await
                                     {
-                                        Ok(_) => {}
+                                        Ok(_) => {
+                                            // TODO: remove this method after we have running risingwave cluster with fast compact algorithm stably for a long time.
+
+                                            if context.storage_opts.check_fast_compaction_result
+                                                && !compact_task.sorted_output_ssts.is_empty()
+                                                && compact_task.task_status() == TaskStatus::Success
+                                                && let Err(e) = check_compaction_result(&compact_task, context.clone()).await
+                                            {
+                                                tracing::error!(
+                "Failed to check fast compaction task {} because: {:?}",
+                compact_task.task_id,
+                e
+            );
```

Comment:
How about panic + print error log + emit a metric here? This can make sure we can easily notice the check failure in test environment.

Reply:
Also, when the check fails, let's log the whole compact task instead of just the task id.

Reply:
If this method failed because of IO, it will print an error log.
If this method failed because of unconsistent check, it will panic at assert code

Reply:
I have add log info for compact task

Reply:
> If this method failed because of unconsistent check, it will panic at assert code

Where is the assert code?